src/iframes/space-bar/spaces-bar.tsx

```diff
@@ -1,58 +1,61 @@
-import React, { useState, useEffect } from "react"
-import { useMount } from "react-use"
-import { Button } from "@netdata/netdata-ui"
+import React, { useState, useEffect } from "react";
```

Comment:
There is no prettierrc in the repo, right? There are a lot of changes applied by your default prettier settings, I think. Let's see if this is the case.

Reply:
done